functions/Get-DbaAgentJobHistory.ps1

```diff
@@ -78,53 +80,89 @@
 			Get-DbaAgentJobHistory -SqlInstance sql2\Inst2K17 -StartDate '2017-05-22' -EndDate '2017-05-23 12:30:00'
 
 			Returns the SQL Agent Job execution results between 2017/05/22 00:00:00 and 2017/05/23 12:30:00 on sql2\Inst2K17.
+	
+		.EXAMPLE 
+	 		Get-DbaAgentJob -SqlInstance sql2016 | Where Name -match backup | Get-DbaAgentJobHistory
+	
+			Gets all jobs with the name that match the regex pattern "backup" and then gets the job history from those. You can also use -Like *backup* in this example.
+	
 	#>
-	[CmdletBinding()]
+	[CmdletBinding(DefaultParameterSetName = "Default")]
 	param (
-		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
+		[parameter(Mandatory, ValueFromPipeline, ParameterSetName = "Server")]
 		[Alias("ServerInstance", "SqlServer")]
 		[DbaInstanceParameter[]]$SqlInstance,
 		[PSCredential][System.Management.Automation.CredentialAttribute()]
 		$SqlCredential,
-		[string]$JobName,
-		[DateTime]$StartDate = '1900-01-01',
+		[string[]]$Job,
+		[DateTime]$StartDate = "1900-01-01",
```

Comment:
The templates are here: https://github.com/sqlcollaborative/dbatools-templates

I have a PR to update some of it that @potatoqualitee needs to approve but just caught we do not have the full TEPP example. I will go update the PR I have on that repo to show it.